### PR TITLE
NCDA for CHW - encounter content

### PR DESCRIPTION
**Pull request by anvmn**
#745 
#746
#747
#748
#749

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-06-19 09-31
Peek 2023-06-19 10-52

**Comment by anvmn** [comment]
Peek 2023-06-19 11-03

**Comment by anvmn** [comment]
Peek 2023-06-19 11-17

**Comment by anvmn** [comment]
Peek 2023-06-19 11-26

**Comment by anvmn** [comment]
Selection_2007

**Comment by anvmn** [comment]
Peek 2023-06-19 11-41

**Comment by anvmn** [comment]
Demo:

Peek 2023-06-19 16-01

**Comment by anvmn** [comment]
Peek 2023-06-19 22-48

File: server/RoboFile.php [hunk 1]
```diff
@@ -146,7 +146,13 @@ public function deployPantheon($branchName = 'master') {
    * @throws \Robo\Exception\TaskException
    */
   public function deployPantheonSync(string $env = 'test', bool $doDeploy = TRUE) {
-    $pantheonName = self::PANTHEON_NAME;
+    if (getenv('PANTHEON_NAME')) {
```
Comment (AronNovak) [inline server/RoboFile.php:149]: Shall we document this somewhere?

**Review by AronNovak — APPROVED** [review]

File: server/RoboFile.php (hunk 1 above)
Comment (anvmn) [reply server/RoboFile.php:149]: @AronNovak We can add it to https://github.com/Gizra/ihangane/wiki/Deployment#project-specific-scripts
We also need `GITHUB_USERNAME` and `GITHUB_ACCESS_TOKEN` for the deploy, so, maybe we should add few words on it as well.

**Pull request merged**